pin-actions.json

```diff
@@ -9,6 +9,10 @@
       "matchDatasources": ["github-tags"],
       "matchPackageNames": ["helm/chart-releaser-action"],
       "allowedVersions": "<=1.5.0"
+    },
+    {
+      "matchManagers": ["nvm"],
+      "rangeStrategy": "replace"
```

Comment:
Should we lock it down to `20`? We don't want to upgrade to other versions atm.

Reply:
I mentioned this in the description -- Renovate wants `engines` driving this and the constraints aren't recommended by them. I think we can allow certain projects to upgrade more aggressively too and we have some teams willing to manage that themselves.